### ISSUE TITLE
Offer an Electron app

### ISSUE THREAD

**Issue by eugenialoli** [photopea/photopea#7369]
I love photopea, but because it runs on a browser, some of the mouse behaviors and ALT/CTRL shortcuts aren't standard (as in, photoshop standard). Some of the mouse/shortcut behaviors of running in a browser are just quirky due to that fact. However, these issues can be overcome if you could offer the app as an Electron app (hopefully for a flat fee with no ads), since Electron allows you to override them. Also, opening massive files might be easier via Electron as RAM won't get capped by the browser.

**Comment by photopea** [comment]
Hi, did you try to press More - Install Photopea? Then, it should work the same as the Electron app.

Opening large files in an Electron app will not work any better as in a browser, because the Electron app still runs a browser.

**Comment by dhenfie** [comment]
I think applications made with Electron run on browsers too

**Comment by DirkBoer** [comment]
I agree that the PWA works really good. Easily the most advanced PWA that I found. 

The best addition for Electron would be for me personally that you have real access to the file system, supporting better opening methods like Open Recent Files, Export to a specific location instead of a download, etc.

I think if you just point to the web version (not local!) it could be a really, really thin layer.

**Comment by eugenialoli** [comment]
You are missing on the whole Linux flathub crowd. Windows and Mac users will always have alternatives, or photoshop itself. The Linux users are starved for a photoshop-like solution, despite Gimp3. Even if you don't offer an electorn app in general, you should at least make one to offer via Flathub. Especially now that Linux userbase is exploding.

**Comment by photopea** [comment]
Photopea should work just as well on Linux as it works on Windows and Macs. What exactly is not working for you when you use Photopea in Linux?

**Comment by eugenialoli** [comment]
I don't think you're understanding me. It's not a matter of not working, of course and it's working! It's a matter of marketing and visibility! Publishing on Flathub automatically puts you on the FRONT ROW of must-have apps for Linux -- for ~50 million users. Exactly because Linux users are hungry for such an app, and they now rely on flathub to extinguish that thirst.

**Comment by photopea** [comment]
I just looked at https://flathub.org/ and I could not find apps like Figma or Youtube. 

Are you sure it has 50 million users? What does it mean? 50 million people open the website every day?

**Comment by eugenialoli** [comment]
Linux has 4% of the desktop OS market, which overall has about 1.2 billion installations worldwide according to stats (you can find that info online). Now, all modern linux distros come with either snap or flatpaks repos pre-installed, and particularly new users (who have been flooding Linux since MS won't support Win11 on older hardware), has been using that as the main way to find and install new apps. Personally, if I was you, and I was trying to get more users, that would be my next step: Flathub and Snap integration. Just for the discoverability aspect.

**Comment by eugenialoli** [comment]
It's for this reason why Boxy SVG has their electron app available on Flathub too, they got an additional 76,000+ downloads from there: https://flathub.org/apps/com.boxy_svg.BoxySVG (and that doesn't count the Snap installations).

**Comment by FabiRich** [comment]
There is a wrapper available on flathub
[Photocrea](https://flathub.org/apps/com.github.vikdevelop.photopea_app)

Found it in the Graphics & Photography category
